### ISSUE TITLE
Installation of 1.0.4 fails with 

### ISSUE THREAD

**Issue by n1k0** [dumbmatter/fakeIndexedDB#6]
Just so you know, installing 1.0.4 currently fails because of a foreign dependency: `phantomjs-prebuilt`.

```
$ npm install fake-indexeddb --save-dev
npm WARN package.json tmp@1.0.0 No description
npm WARN package.json tmp@1.0.0 No repository field.
npm WARN package.json tmp@1.0.0 No README data
npm WARN deprecated graceful-fs@2.0.3: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
 
> phantomjs-prebuilt@2.1.5 install /home/niko/tmp/node_modules/fake-indexeddb/node_modules/phantomjs-prebuilt
> node install.js

module.js:339
    throw err;
    ^

Error: Cannot find module './util/assign'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/niko/tmp/node_modules/fake-indexeddb/node_modules/fs-extra/lib/index.js:1:76)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
npm ERR! Linux 4.2.0-34-generic
npm ERR! argv "/home/niko/.nvm/versions/node/v4.2.1/bin/node" "/home/niko/.nvm/versions/node/v4.2.1/bin/npm" "install" "fake-indexeddb" "--save-dev"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE

npm ERR! phantomjs-prebuilt@2.1.5 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the phantomjs-prebuilt@2.1.5 install script 'node install.js'.
npm ERR! This is most likely a problem with the phantomjs-prebuilt package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls phantomjs-prebuilt
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/niko/tmp/npm-debug.log
```

I'm filing this here just so you know about the issue and can track it until it's fixed.

**Comment by dumbmatter** [comment]
Thanks! I hadn't noticed this. And it's funny, earlier today I had a similar problem with another package at my day job, and I spent a long time debugging it before I found the solution.

The issue is, shrinkwrap files created with npm 3 sometimes fail when used with npm 2, and vice versa. And this shrinkwrap file, which I created using npm 3, does indeed not work with npm 2.

Fortunately in this case, if I create the shrinkwrap file with npm 2, it does seem to work with npm 3. So I released v1.0.5 with a new shrinkwrap file made with npm 2.

God, I hate npm sometimes...

The one thing I don't understand is why my CI passed for v1.0.4 because that did include testing with npm 2. When I reran CI it failed. Weird.
